src/event.js

```diff
@@ -746,6 +746,12 @@ jQuery.each( { focus: "focusin", blur: "focusout" }, function( type, delegateTyp
 			return true;
 		},
 
+		// Suppress native focus or blur as it's already being fired
+		// in leverageNative.
+		_default: function() {
```

Comment:
Yeah I hadn't thought about that. The `_default` action happens at the very very very end of this process so as long as `leverageNative` always does the work for us earlier we don't need to do it here.